examples/Joints/Ingen/pinjoint.py

```diff
@@ -0,0 +1,16 @@
+from sympy import Symbol
+from pydy.bodies import Body
+from pydy.joints import PinJoint
+from pydy.joints_method import JointsMethod
+
+parent = Body('parent')
+child = Body('child')
+
+gravity = Symbol('gravity')
+child.add_force((0, child.get_mass() * gravity, 0), child.get_masscenter())
```

Comment:
1. But adding the force to the child's y direction is incorrect.
2. add_force is not needed, because if `.forces` is just a list, then all you need to do is `.forces.append(...)`

`get_masscenter` should use @property instead.